apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/BeneficiaryTransformer.java

```diff
@@ -258,8 +258,29 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
       patient.addExtension(
           TransformerUtils.createExtensionDate(
               CcwCodebookVariable.RFRNC_YR, beneficiary.getBeneEnrollmentReferenceYear()));
+
+      addMedicaidDualEligibility(patient, beneficiary);
```

Comment:
Why is this part of this PR?

Reply:
just making it uniform instead of one big function